prisma/schema.prisma

```diff
@@ -11,16 +11,22 @@ datasource db {
 }
 
 model users {
-  id           String    @id @default(uuid()) @db.Uuid
-  userName     String    @unique
-  password     String
-  firstName    String
-  lastName     String
-  discount     Boolean   @default(false)
-  freeShipping Boolean   @default(false)
-  dateCreated  DateTime  @default(now()) @db.Date
-  orders       orders[]
-  reviews      reviews[]
+  id            String    @id @default(uuid()) @db.Uuid
+  userName      String    @unique
+  password      String
+  firstName     String
+  lastName      String
+  email         String?    @unique
+  birthDate     String    @default("MM-DDTHH-YYYY")
+  streetAddress String?
+  city          String?
+  state         String?
+  zipCode       String?
```

Comment:
Could you change this to a number instead of a string.

Reply:
I changed it, I git pushed but it's not showing me an option to do pull request of changes